src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngKey.Create.cs

```diff
@@ -149,6 +159,114 @@ private static void InitializeKeyUiPolicyProperties(SafeNCryptKeyHandle keyHandl
                 }
             }
         }
+
+        /// <summary>
+        ///     Return a CngProperty representing a named curve.
+        /// </summary>
+        internal static CngProperty GetPropertyFromNamedCurve(ECCurve curve)
```

Comment:
The remaining helper methods in this file feel like they better belong in the new EC partial file...